gapis/service/service.proto

```diff
@@ -397,6 +397,9 @@ message ErrDataUnavailable {
   stringtable.Msg reason = 1;
   // If true, then making the same request at a later time may result in data.
   bool transient = 2;
+  // If true, draw call specific data was requested for a command range not
+  // containing any draw calls.
+  bool not_a_draw_call = 3;
```

Comment:
Yes, sorry for the delay, here is my response:

It's fairly common practice for servers to give both a human readable error message along with machine readable info/codes. In this circumstance, I want to have three different states: 1) success, 2) not a draw call, 3) error. 2) shows a specific message without an error icon and feel: "Select a draw call". From the client's perspective, the no draw call error is expected, but unavoidable, that should be treated differently than any other error.

In general, I think it makes sense to give clients extra info regarding errors so they can attempt to handle them in their own unique way (for example by given the user information about how to resolve it, as is the case here), and treat the server generated messages as a fallback/general case.